documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='ref-cruise-control-configuration-{context}']
+= Cruise Control configuration
+
+The `config` property in `Kafka.spec.cruiseControl` contains configuration options as keys with values as one of the following JSON types:
+
+* String
+* Number
+* Boolean
+
+NOTE: Strings that look like JSON or YAML will need to be explicitly quoted.
+
+You can specify and configure all the options listed in the "Configurations" section of the {CruiseControlConfigDocs}, apart from those managed directly by {ProductName}.
+Specifically, you *cannot* modify configuration options with keys equal to or starting with one of the following strings:
+
+* `bootstrap.servers`
+* `zookeeper.`
+* `ssl.`
+* `security.`
+* `failed.brokers.zk.path`
+* `webserver.http.port`
+* `webserver.http.address`
+* `webserver.api.urlprefix`
+* `metric.reporter.sampler.bootstrap.servers`
+* `metric.reporter.topic`
+* `metric.reporter.topic.pattern`
+* `partition.metric.sample.store.topic`
+* `broker.metric.sample.store.topic`
+* `capacity.config.file`
+* `skip.sample.store.topic.rack.awareness.check`
+* `cruise.control.metrics.topic`
+* `sasl.`
+
```

Comment:
We'll need to update this for the ssl ones too.

Reply:
I am going to enable ssl configurations until I secure the communication between Cruise Control and Kafka brokers with SSL

Reply:
OK